buildSrc/src/main/java/Config.kt

```diff
@@ -3,8 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import org.gradle.api.Project
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
 
 object Config {
+
+    @JvmStatic
+    private fun generateDebugVersionName(): String {
+        val today = Date()
+        // Append the year (2 digits) and week in year (2 digits). This will make it easier to distinguish versions and
+        // identify ancient versions when debugging issues. However this will still keep the same version number during
+        // the week so that we do not end up with a lot of versions in tools like Sentry. As an extra this matches the
+        // sections we use in the changelog (weeks).
+        return SimpleDateFormat("1.0.yyww", Locale.US).format(today)
```

Comment:
From personal experience, having the yy-mm-dd (minutes/seconds are optional) for a debug build is helpful to avoid user error when you think you're running a new debug but somehow it wasn't deployed to your device; week number is hard to grok.

Same would go for Fenix.

Reply:
I filed a followup for this https://bugzilla.mozilla.org/show_bug.cgi?id=1803686